test/helpers/http.ts

```diff
@@ -20,7 +20,7 @@ type RequestShape = {
 export async function expectRequest(request: Request, shape: RequestShape) {
 
   expect(request.method).to.equal(shape.method)
-  expect(new URL(request.url).pathname).to.equal(shape.path)
+  if (shape.path !== undefined) expect(new URL(request.url).pathname).to.equal(shape.path)
```

Comment:
Please wrap `if` then in curly braces for consistency.

Were you around for the Apple `goto fail` bug? https://www.imperialviolet.org/2014/02/22/applebug.html I am in the camp that curly braces would have helped, but unit tests definitely would have helped this case more so.

Reply:
I _only_ ever do this if it's on the same line, but happy to change this. There's probably an eslint rule for this too so ill take a look at that.